extension/src/fileSystem/index.ts

```diff
@@ -141,6 +141,9 @@ export const openFileInEditor = async (filePath: string) => {
   return document
 }
 
+export const openImageFileInEditor = async (imagePath: string) =>
+  await commands.executeCommand('vscode.open', Uri.file(imagePath))
```

Comment:
[Q] Should this be open to the side? (`explorer.openToSide`). Definitely not a statement. Not sure of the best UX.

Reply:
I thought it might be a little small for images on a 13" laptop, but it does look good. I like the fact that you don't lose context when opening to the side, you can also open multiple images without going back and forth. I'll make the change.